tests/chainerx_tests/unit_tests/routines_tests/test_math.py

```diff
@@ -1780,7 +1810,7 @@ def func(self, xp, a):
         return xp.sqrt(a)
 
 
-_trignometric_hyperpolic_params = \
+_trigonometric_hyperpolic_params = \
```

Comment:
A nitpick but should it be "hyperbolic"?

Reply:
Fixed.